brainframe/api/stubs/base_stub.py

```diff
@@ -58,13 +58,13 @@ def _get_json(self, api_url, timeout, params=None) -> Tuple[Any, dict]:
             return json.loads(resp.content), resp.headers
         return None, resp.headers
 
-    def _put_json(self, api_url, timeout, json_data):
+    def _put_json(self, api_url, timeout, json_data) -> Optional[Any]:
```

Comment:
Aren't `Optional[Any]` and `Any` equivalent?
`None` would satisfy `Any`

Reply:
It would be nice if https://github.com/python/typing/issues/182 was solved

Reply:
Good point